SuperCalculator/src/supercalculator/F6.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package supercalculator;
+
+/**
+ *
+ * @author za
+ */
+public class F6 {
```

Comment:
Please consider renaming the class to your [function name] since the name doesn't explicitly explain what is the function